packages/store/addon/-private/system/store.ts

```diff
@@ -1270,7 +1271,7 @@ const Store = Service.extend({
     return _findHasMany(adapter, this, internalModel, link, relationship, options);
   },
 
-  _findHasManyByJsonApiResource(resource, parentInternalModel, relationshipMeta, options) {
+  _findHasManyByJsonApiResource(resource, parentInternalModel, relationshipMeta, options): Promise<unknown> {
```

Comment:
Sometimes we have a `Promise<any>` and sometimes `Promise<unknown>`, I remember running into this issue as well, also related to RSVP vs native Promise or smth. Any ideas/maybe consult @mike-north

Reply:
Mike and I investigated this and discovered the issue is RSVP typings, he PR'd a fix yesterday.

Reply:
(We want to use `Promise<unknown>`)